test_data/products.json

```diff
@@ -0,0 +1,11 @@
+{
+    "product1": {
```

Comment:
I'm not a fan of variable numbering. During test development, it tells me nothing, I have to check the contents, and in case of deleting a product, all variables have to be renumbered.